### DISCUSSION TITLE
How to retrieve access logs

### DISCUSSION THREAD

**Discussion by Resousse** [vercel/community#744] [Help] [answered]
Hello,

I don't find anything in document about this, but is it possible to retrieve access logs (in a Apache/Nginx format), for the pure html part (not serverless)?

Thanks

**Comment by amyegan** [comment, marked as answer]
Hi @Resousse. Vercel, by default, does not store logs from static assets and Serverless Functions. It is possible to [use Log Drain Integrations](https://vercel.com/support/articles/how-do-i-store-logs-on-vercel) to store log details. Vercel also offers an [Analytics](https://vercel.com/docs/concepts/analytics) feature to track how well your application is performing in a technical sense.
